src/sentry/monitors/validators.py

```diff
@@ -258,3 +259,12 @@ def validate(self, attrs):
             del attrs["monitor_config"]
 
         return attrs
+
+
+class MonitorAlertRuleTargetValidator(serializers.Serializer):
+    target_identifier = serializers.IntegerField()
```

Comment:
We're not able to re-use any of the validation logic from the alert rule itself are we?

Reply:
Would also be nice to add `help_text` fields to these to improve the generated documentation.